### ISSUE TITLE
proposal: use details, summary tag to hide too match resource on comment

### ISSUE THREAD

**Issue by guitarrapc** [pulumi/actions#26]
## TL;DR

When set `COMMENT_ON_PR: 1` for GitHub Actions environment, action bot will comment changes to PR comment. However actions comment may become too long for many resources and it occupy screen visibility for continuous commit.  
I would propse using `details`/`summary` HTML tag to hide resources result by default.

#### :tropical_drink: `pulumi --non-interactive preview --refresh`

```
Previewing update (master):
    pulumi:pulumi:Stack aws-sandbox-master  running 'dotnet build -nologo .'
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master  Welcome to .NET Core 3.1!
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master    Restore completed in 6.81 sec for /github/workspace/pulumi/aws-sandbox/infra.csproj.
    pulumi:pulumi:Stack aws-sandbox-master    AwsSandbox -> /github/workspace/pulumi/aws-sandbox/bin/Debug/netcoreapp3.1/infra.dll
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master      0 Warning(s)
    pulumi:pulumi:Stack aws-sandbox-master  'dotnet build -nologo .' completed successfully
[resource plugin aws-2.8.0] installing
... // many logs....
 ~  pulumi:pulumi:Stack aws-sandbox-master refreshing 'dotnet build -nologo .' completed successfully
 ~  pulumi:pulumi:Stack aws-sandbox-master refreshing 'dotnet build -nologo .' completed successfully
... // depends on count of resource. it may over 200 lines....!!
Diagnostics:
  pulumi:pulumi:Stack (aws-sandbox-master):
    The launch profile "(Default)" could not be applied.
    A usable launch profile could not be located.
 

Permalink: https://app.pulumi.com/YOUR_ORG/aws-sandbox/master/previews/xxxxxxxxxxxxxxxxxxxxxxxxx
```

#### :tropical_drink: `pulumi --non-interactive preview --refresh`

<details>
<summary>Details (Click me)</summary>
<pre><code>
Previewing update (master):
    pulumi:pulumi:Stack aws-sandbox-master  running 'dotnet build -nologo .'
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master  Welcome to .NET Core 3.1!
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master    Restore completed in 6.81 sec for /github/workspace/pulumi/aws-sandbox/infra.csproj.
    pulumi:pulumi:Stack aws-sandbox-master    AwsSandbox -> /github/workspace/pulumi/aws-sandbox/bin/Debug/netcoreapp3.1/infra.dll
    pulumi:pulumi:Stack aws-sandbox-master  
    pulumi:pulumi:Stack aws-sandbox-master      0 Warning(s)
    pulumi:pulumi:Stack aws-sandbox-master  'dotnet build -nologo .' completed successfully
[resource plugin aws-2.8.0] installing
...
 ~  pulumi:pulumi:Stack aws-sandbox-master refreshing 'dotnet build -nologo .' completed successfully
 ~  pulumi:pulumi:Stack aws-sandbox-master refreshing 'dotnet build -nologo .' completed successfully
...
Diagnostics:
  pulumi:pulumi:Stack (aws-sandbox-master):
    The launch profile "(Default)" could not be applied.
    A usable launch profile could not be located.
 

Permalink: https://app.pulumi.com/YOUR_ORG/aws-sandbox/master/previews/xxxxxxxxxxxxxxxxxxxxxxxxx
</code>
</pre>
</details>

## Effected code line

> https://github.com/pulumi/actions/blob/v0.1/entrypoint.sh#L99-L102

## NOTE

I feel it friendly if comment describe summary pf changes like `xxx new, yyyy update, zzzz remove`.
